recipe/meta.yaml

```diff
@@ -8,12 +8,15 @@ package:
 source:
   url: https://github.com/ofiwg/libfabric/releases/download/v{{ version }}/libfabric-{{ version }}.tar.bz2
 # sha256: 485e6cafa66c9e4f6aa688d2c9526e274c47fda3a783cf1dd8f7c69a07e2d5fe  # 1.20.0
-  sha256: 485e6cafa66c9e4f6aa688d2c9526e274c47fda3a783cf1dd8f7c69a07e2d5fe  # 1.15.2
+  sha256: b8839e56d80470a917453a7d8ad9cb717f6683fee28cf93de5f3a056ed4f04c8  # 1.19.1
+# sha256: 912fb7c7b3cf2a91140520962b004a1c5d2f39184adbbd98ae5919b0178afd43  # 1.18.0
+# sha256: ac104b9d6e3ce8bda6116329e3f440b621d85602257b3015116ca590f65267d2  # 1.16.0
+# sha256: 485e6cafa66c9e4f6aa688d2c9526e274c47fda3a783cf1dd8f7c69a07e2d5fe  # 1.15.2
 # sha256: fc261388848f3cff555bd653f5cb901f6b9485ad285e5c53328b13f0e69f749a  # 1.14.0
 
 build:
   number: {{ build }}
-  skip: true  # [win]
+  skip: true  # [win or osx]
 
   run_exports:
     - {{ pin_subpackage('libfabric', max_pin='x') }}
```

Comment:
If we're right about the ABI, this should be `libfabric >=1.14,<2.0`. Are you able to test this compatibility without the run_exports?